core/src/main/java/bisq/core/api/CoreTradesService.java

```diff
@@ -292,7 +298,10 @@ Trade getTrade(String tradeId) {
     List<TradeModel> getOpenTrades() {
         coreWalletsService.verifyWalletsAreAvailable();
         coreWalletsService.verifyEncryptedWalletIsUnlocked();
-        return tradeManager.getTrades().stream().map(t -> (TradeModel) t).collect(Collectors.toList());
+        return tradeManager.getTrades().stream()
+                .map(t -> (TradeModel) t)
```

Comment:
Why do you need to map it here?

Reply:
It is something @xyz123 asked for in his review:  "why isn't the mapping consistent between return types for `List<TradeModel> getOpenTrades()` and ` List<TradeModel> getTradeHistory(...)`"?   I did intend make the return types consistent, but overlooked it.

Reply:
I don't mean so much about the return types of the method, but rather the fact, that in this case it is unnecessary to cast.

Reply:
I removed two casts in https://github.com/bisq-network/bisq/pull/5976/commits/820642516a64ed85b0b365dc8c22305d45c54951, but the mapping in line 303 had to stay.